blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -41,6 +41,8 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Remove deprecated nGram, edgeNGram, htmlStrip token filter names in favour for ngram, edge_ngram, and html_strip
```

Comment:
Please use past tense, surround nGram etc with double backticks, respect the 79 char limit per line and add a `.` and the end of a sentence.

Reply:
`html_strip` is a char filter.